ytxp-plutarch/src/Cardano/YTxP.hs

```diff
@@ -38,49 +40,77 @@ data YTxPParams = YTxPParams
   deriving stock (Show, Generic)
   deriving anyclass (ToJSON, FromJSON)
 
-linker :: Linker YTxPParams (ScriptExport YTxPParams)
-linker = do
+validatorLinker :: Linker YTxPParams (ScriptExport YTxPParams)
+validatorLinker = do
   info <- getParam
 
   yieldingValidator <-
     fetchTS
       @'ThreeArgumentScript
       @'[CurrencySymbol]
-      "djed:yieldingV"
+      "ytxp:yieldingV"
```

Comment:
I personally find it a bit strange that sometimes we abbreviate `validator` with `v` and some other times it's the full. Also I think it's not very consistent:
- `validatorLinker`
- `mpLinker`



I'd probably prefer either:
- `vLinker`
- `mpLinker`
or
- `validatorLinker`
- `mintingPolicyLinke`

But I understand it's a stylistic thing and I'm approving this anyway, feel free to update the PR or ignore this :relaxed: